daemon/cmd/debuginfo.go

```diff
@@ -48,12 +50,11 @@ func (h *getDebugInfo) Handle(params restapi.GetDebuginfoParams) middleware.Resp
 	dr.CiliumMemoryMap = memoryMap(os.Getpid())
 
 	dr.EnvironmentVariables = []string{}
-	// FIXME
-	// for _, k := range Vp.AllKeys() {
-	// 	// Assuming we are only getting strings
-	// 	v := fmt.Sprintf("%s:%s", k, Vp.GetString(k))
-	// 	dr.EnvironmentVariables = append(dr.EnvironmentVariables, v)
-	// }
+
+	for k, v := range d.settings {
+		envValue := fmt.Sprintf("%s:%s", k, v)
+		dr.EnvironmentVariables = append(dr.EnvironmentVariables, envValue)
+	}
```

Comment:
I see you've resolved it here, but can we just remove the extra diff from the previous commit?

Reply:
Unfortunately we can't do that because of the removal of the global `Vp` variable (wouldn't compile).
What do you think about adding a more extended comment (for both the controlplane tests and the debuginfo handler) instead of a bare `// FIXME`, reporting what I already pointed out in the commit description?

The alternative would be to squash all the commits, but that would produce something too large, IMHO.